app/DoctrineMigrations/Version20200924094631.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Contributor categories as a simple array.
+ */
+final class Version20200924094631 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->abortIf('mysql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE contributor ADD categories LONGTEXT NULL COMMENT \'(DC2Type:simple_array)\'');
```

Comment:
I thought we discussed having a `hashtag` table with all the hashtags, with a relation to `contributor`, so that we can easily ask the API for all contributors with a specific hashtag (https://notices.bulles.fr/api/v3/contributors?hashtags[]=someHashtag) ?